src/main/java/com/smartcar/sdk/data/BatchResponse.java

```diff
@@ -0,0 +1,105 @@
+package com.smartcar.sdk.data;
+
+import com.smartcar.sdk.SmartcarException;
+
+import org.joda.time.DateTime;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.GsonBuilder;
+import org.apache.commons.text.CaseUtils;
+import java.util.HashMap;
+import java.util.Iterator;
+
+public class BatchResponse extends ApiData {
+  private HashMap<String, JsonObject> responseData = new HashMap<>();
```

Comment:
Use the interface here:
```suggestion
  private Map<String, JsonObject> responseData = new HashMap<>();
```

You'll also need to import `java.util.Map`